spec.go

```diff
@@ -438,6 +438,9 @@ func createCgroupConfig(name string, spec *specs.LinuxRuntimeSpec, devices []*co
 	if err != nil {
 		return nil, err
 	}
+	if spec.Linux.CgroupsPath != "" {
+		name = spec.Linux.CgroupsPath
+	}
```

Comment:
@ashahab-altiscale I think the problem here is `name` should not identify a cgroup, because it should just be the name, not the full path. 
In the code, cgroup path can be affected by `parent` besides `name`, though it won't happen in real life for now because we only support `parent` for Docker and Docker don't use `cgroupsPath`. But that'll be tricky if we want to expand these features in the future.